src/ddmd/objc.d

```diff
@@ -117,126 +117,140 @@ struct ObjcSelector
     }
 }
 
-struct Objc_ClassDeclaration
-{
-    // true if this is an Objective-C class/interface
-    bool objc;
+private __gshared Objc _objc;
 
-    // MARK: Objc_ClassDeclaration
-    extern (C++) bool isInterface()
-    {
-        return objc;
-    }
+Objc objc()
+{
+    return _objc;
 }
 
-struct Objc_FuncDeclaration
+interface Objc
```

Comment:
I did have a problem with this PR https://github.com/dlang/dmd/pull/6625 when building DMD with `-unittest`, which it will do by default when building DMD with `DEBUG=1`. But as far as I can the auto tester is building a release build and not a debug build, i.e. `-unittest` is not supplied.